maintainers/maintainer-list.nix

```diff
@@ -5807,6 +5807,12 @@
     githubId = 1151264;
     name = "Sebastian Graf";
   };
+  shahrukh330 = {
+    email = "shahrukh330@gmail.com";
+    github = "shahrukh330";
+    githubId = 1588288;
+    name = "Shahrukh Khan";
+  };
```

Comment:
@shahrukh330 please put this in a separate commit. Your PR should have one commit for the new maintainer, and another for the qds_sdk init (current commit).

Reply:
Reverted the previous commit and added two separate commits.

Reply:
Please force-push to remove the previous commit and the revert. The PR should only contain two commits:
- maintainers: add shahrukh330
- pythonPacakges.qds_sdk: init at 1.12.0

Thank you!